onnxruntime/core/framework/execution_frame.cc

```diff
@@ -86,13 +86,54 @@ Status IExecutionFrame::GetOrCreateNodeOutputMLValue(int index, const TensorShap
                     " Requested shape:", shape ? shape->ToString() : "null");
       }
     } else {
+      VerifyOutputSizes(output_index, node, shape);
       status = CreateNodeOutputMLValueImpl(*p_ort_value, ort_value_idx, shape, nnz);
     }
   }
 
   return status;
 }
 
+void IExecutionFrame::VerifyOutputSizes(int output_index, const onnxruntime::Node& node,
```

Comment:
onnxruntime:: is redundant. Several other places as well.

Reply:
Removed.